example/README.rst

```diff
@@ -1,14 +1,25 @@
 Example Project
 ===============
 
-Use Python 3.11 to set up and run with these commands:
+Use Python 3.11 to set up with these commands:
 
 .. code-block:: sh
 
    python -m venv .venv
    source .venv/bin/activate
    python -m pip install -U pip wheel
    python -m pip install -r requirements.txt -e ..
+
+Run the sync WSGI server with:
+
+.. code-block:: sh
+
+   DEBUG=1 python manage.py runserver --noasgi
+
+Run the async ASGI server with:
+
+.. code-block:: sh
```

Comment:
adding ASGI to the example project was key to testing this... did you not see the example project before? You tested only in your own project?

Reply:
If I remember correctly, it was just easier for me to simply add this to my own project than to run the example project. But as this was a couple months ago, it’s also very well possible that I just didn’t see it.